### PR TITLE
WT-10340 Add stats for sessions that did not run a sweep for a while.

### PR DESCRIPTION
**Pull request by pmacko86**
It also adds a warning for sessions that did not sweep for more than an hour.

### CODE REVIEW

File: src/conn/conn_sweep.c [hunk 1]
```diff
@@ -261,6 +261,67 @@ __sweep_server_run_chk(WT_SESSION_IMPL *session)
...
+        if (!s->active)
+            continue;
+        if (F_ISSET(s, WT_SESSION_INTERNAL))
+            continue;
+
+        last_cursor_big_sweep = s->last_cursor_big_sweep;
+        last_sweep = s->last_sweep;
+
+        last = last_cursor_big_sweep;
+        if (last_sweep > last)
```
Comment (ddanderson) [inline src/conn/conn_sweep.c:291]: I wonder if want this inequality reversed?  E.g. last_cursor_big_sweep is timestamp 10000 and the last_sweep is 50000 and now is 51000.  Maybe last_sweep is keeping up, but the cursor sweep is not.  It doesn't look like this catches that?  Or am I missing something?

These two sweeps clean up two data structures.  Both of the structures separately may hold references to dhandles.

File: src/conn/conn_sweep.c [hunk 2]
```diff
@@ -261,6 +261,67 @@ __sweep_server_run_chk(WT_SESSION_IMPL *session)
...
+        if (last == 0)
+            continue;
+
+        /*
+         * Check if the session did not run a sweep in 5 minutes. Handle the issue only once per
+         * violation.
+         */
+        if (last + 5 * 60 < now) {
+            if (!s->sweep_warning_5min) {
+                s->sweep_warning_5min = 1;
```
Comment (ddanderson) [inline src/conn/conn_sweep.c:302]: I think we still want a statistic for this, even if we don't report it as a warning.  Sorry if I didn't communicate that.

**Review by ddanderson — COMMENTED** [review]
@pmacko86, sorry for the late review.  I have a couple comments/questions.

File: src/conn/conn_sweep.c (hunk 2 above)
Comment (pmacko86) [reply src/conn/conn_sweep.c:302]: Oh that's right, sorry. I was deleting the warning and took down the stats for it as well; I'll put it back.

File: src/conn/conn_sweep.c (hunk 1 above)
Comment (pmacko86) [reply src/conn/conn_sweep.c:291]: I didn't realize that these were two different processes, in which case we should then take the earlier non-zero timestamp.

**Comment by ddanderson** [comment]
@pmacko86, when you are ready for a review again, click the circular arrows next to my name on the upper right of this page.  Perhaps you already knew this.  It's part of our process, it may not be a part of everyone's.

**Comment by pmacko86** [comment]
@ddanderson, thank you for the feedback. I added the 5-min stat, and I now take the smaller of last_cursor_big_sweep and last_sweep. Hopefully this fixes the issues.

**Review by ddanderson — APPROVED** [review]
@pmacko86 Looks good!  I suggested one comment improvement.  You can merge without another review.

**Pull request merged**